.github/workflows/main.yml

```diff
@@ -85,3 +85,28 @@ jobs:
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
           password: ${{ secrets.PYPI_API_TOKEN }}
+
+
+  trigger_endless-key-app:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/') || startsWith(inputs.tagname, 'v')
+    needs: [build_apps-bundle, build_kolibri-explore-plugin_whl]
+
+    steps:
+      - name: Get tag name as the release version when at a tag
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          echo "VERSION=${{ github.ref_name }}" >> $GITHUB_ENV
+
+      - name: Get tag name as the release version from input
+        if: ${{ !startsWith(github.ref, 'refs/tags/') && startsWith(inputs.tagname, 'v') }
+        run: |
+          echo "VERSION=${{ inputs.tagname }}" >> $GITHUB_ENV
```

Comment:
Side note: there is a "friendly" version name that changes with each major version. It lives in `kolibri_explore_plugin/VERSION`. Current is "Ladybird".

We use that to form the Android version: Ladybird 6.2-329

where:
- "Ladybird" comes from the VERSION file
- 6.2 is the tag minus the patch (we don't use patch number)
- 329 is the Android specific version

It would be great if the Windows app can follow the same versioning!

Reply:
Microsoft defines MSIX/APPX's version number as `Major.Minor.Build.Revision`.
https://github.com/microsoft/WindowsAppSDK/blob/main/specs/Deployment/MSIXPackageVersioning.md#122-version-number--msix

* We can sync the Major and Minor as kolibri-explore-plugin's version minus the patch
* Build: increase number simply for Windows app build
* Revision: is `0`

However, there is no slot for the VERSION name, "Ladybird".

But, synchronizing the number part into APPX's version number is a good suggestion.

To keep this PR moving, I split this idea as ticket https://github.com/endlessm/endless-key-app/issues/119

Reply:
Then, this PR can focus on the webhook.

Reply:
Pushed commit ("CI: Sync the tag as kolibri-explore-plugin's major & minor version") into PR [CI: Add bumpversion.yml workflow as a webhook target](https://github.com/endlessm/endless-key-app/pull/118).

Reply:
@starnight I wouldn't tie the kolibri-explore-plugin version to the Windows app version. At least on Android there are cases where the app's wrapping code needs a fix that should be released without any changes in the plugin.

In Android the real version (versionCode) is just an integer that must always increase. The Jenkins CI is automatically bumping that version number. Then there is a version name (versionName) that can be whatever string, that's where we put "Ladybird 6.2-329". https://developer.android.com/studio/publish/versioning#versioningsettings

If Windows uses semantic versioning, could this CI automatically increase the patch or minor version?